### DISCUSSION TITLE
Creating Webhooks in Vercel

### DISCUSSION THREAD

**Discussion by Patrikur** [vercel/community#2769] [Help] [answered]
### Page to Investigate

https://vercel.com/docs/integrations/webhooks-overview

### Steps to Reproduce

Hi,

I am trying to create a webhook in Vercel which gets triggered by a new post that is created in WordPress headless CMS.
The problem is after your current updates, the documentation tells briefly about the Endpoint URL that needs to be inserted, but not really which one is needed. I have several endpoints from which data comes into the project. Shall I use all of them? Could someone please elaborate this a little more detailed what is exactly needed here?

Cheers in advance for the help!

**Comment by smaeda-ks** [comment]
@Patrikur You will only have one unified endpoint that listens to all Webhook events, and this endpoint parses the event payload (POST body) and decide what to do. The payload type is documented here: https://vercel.com/docs/integrations/webhooks-overview/webhooks-api

**Reply by daniel-bergmann** [reply]
@smaeda-ks  could you give us a step by step example how we could achieve this?

**Comment by amyegan** [comment, marked as answer]
Are you using [Vercel Webhooks](https://vercel.com/docs/integrations/webhooks-overview) to subscribe to deployment events, or are you using [Deploy Hooks](https://vercel.com/docs/concepts/deployments/deploy-hooks) to trigger a new deployment via CMS after making content updates?

**Reply by daniel-bergmann** [reply]
@amyegan, this clue solved the issue, deploy hooks can be found under `Project Name/Settings/Git/Deploy Hooks`

**Reply by Patrikur** [reply]
@amyegan Thanks so much, this was the answer we were looking for :)
